libraries/libfc/include/fc/crypto/private_key.hpp

```diff
@@ -53,7 +53,7 @@ namespace fc { namespace crypto {
          storage_type _storage;
 
          private_key( storage_type&& other_storage )
-         :_storage(forward<storage_type>(other_storage))
+            :_storage(std::move(other_storage))
```

Comment:
`other_storage` is trivially copiable, I think just remove `std::move`.

Reply:
Yes, thanks!